xbout/tests/test_boutdataarray.py

```diff
@@ -3,13 +3,11 @@
 import dask.array
 import numpy as np
 import numpy.testing as npt
-from pathlib import Path
 
 import xarray as xr
 import xarray.testing as xrt
 from xarray.core.utils import dict_equiv
 
-from xbout.tests.test_load import bout_xyt_example_files, create_bout_ds
```

Comment:
Ah, looks like this import was flagged as unused because it only appears as a fixture. Moving it to `conftest.py` is probably the best way to fix this